localstack/testing/pytest/fixtures.py

```diff
@@ -328,16 +330,6 @@ def sqs_queue(sqs_create_queue):
     return sqs_create_queue()
 
 
-@pytest.fixture
-def sqs_queue_arn(aws_client):
-    def _get_arn(queue_url: str) -> str:
-        return aws_client.sqs.get_queue_attributes(QueueUrl=queue_url, AttributeNames=["QueueArn"])[
-            "Attributes"
-        ]["QueueArn"]
-
-    return _get_arn
-
-
```

Comment:
There was a fixture and a utility (`localstack.utils.aws.arns.sqs_queue_arn`) with the same name that caused confusion and silent overwrite in some instances. The fixture is removed in favour of the utility.